From 5155e4015a08060c3f5b34084c81929b219ded7c Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Thu, 15 Nov 2018 19:20:12 -0700
Subject: [PATCH 6/8] klte-common: restorecon I/O scheduler tunables before
 touching them

* These directories and nodes get dynamically created in a way that
  doesn't result in them being labeled correctly by default. Do this
  so that init can actually write to these nodes.

Change-Id: I493a9bb0c1bd2ea83a815e07169b8b689cf76a5d
---
 rootdir/etc/init.qcom.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rootdir/etc/init.qcom.rc b/rootdir/etc/init.qcom.rc
index 58ff196..93fd5aa 100644
--- a/rootdir/etc/init.qcom.rc
+++ b/rootdir/etc/init.qcom.rc
@@ -33,6 +33,7 @@ on early-init
     chmod 0755 /sys/kernel/debug
 
     # Increase max readahead size to 512 KB
+    restorecon --recursive /sys/block/mmcblk0/queue
     write /sys/block/mmcblk0/queue/read_ahead_kb 512
     write /sys/block/mmcblk0/queue/scheduler noop
 
-- 
2.17.1

